scripts/eslint-rules/__tests__/no-production-logging-test.internal.js

```diff
@@ -18,28 +18,35 @@ ruleTester.run('no-production-logging', rule, {
     {
       code: `
         if (__DEV__) {
-          warning(test, 'Oh no');
+          console.error('Oh no');
         }
       `,
     },
     {
       code: `
         if (__DEV__) {
-          warningWithoutStack(test, 'Oh no');
+          console.error('Hello %s', foo)
         }
       `,
     },
     {
       code: `
         if (__DEV__) {
-          lowPriorityWarning(test, 'Oh no');
+          console.error('Hello %s %s', foo, bar)
```

Comment:
This change doesn't look equivalent. Shouldn't low-pri have mapped to `console.warn`?

Reply:
That’s just git diff being funky. I added a few cases here.